ui/app/templates/components/secret-list-header.hbs

```diff
@@ -1,7 +1,7 @@
 {{#with (options-for-backend @model.engineType) as |options|}}
   <PageHeader as |p|>
     <p.top>
-      <KeyValueHeader @baseKey={{this.baseKey}} @path="vault.cluster.secrets.backend.list" @root={{@backendCrumb}}>
+      <KeyValueHeader @baseKey={{@baseKey}} @path="vault.cluster.secrets.backend.list" @root={{@backendCrumb}}>
```

Comment:
Ah, probably a backport to 1.9 because I believe in 1.9 I glimmerized this component and this would be a glimmer issue. Nice catch!

Reply:
@Monkeychip - from what I could tell this behavior has been happening since before 1.7? But I'll reconfirm next week